app/controllers/concerns/api/v2/taxonomies_controller.rb

```diff
@@ -2,6 +2,7 @@ module Api::V2::TaxonomiesController
   extend ActiveSupport::Concern
 
   included do
+    before_filter :rename_config_template
```

Comment:
Add `:only ` to run it only on the actions where you could be passing the parameter. Also it looks like you can probably avoid going in the method with an `:if` here `  if params[taxonomy_single] && params[taxonomy_single][:config_template_ids].present?`